### DISCUSSION TITLE
Realized and Unrealized Returns

### DISCUSSION THREAD

**Discussion by iGaneshKumar** [ghostfolio/ghostfolio#3579] [Ideas]
Hi,
Let's say I've invested a lump sum in a mutual fund. After some months, the value increased, and I exited that investment with a return. Later, I reinvested in the same mutual fund and haven't exited yet. The absolute net performance shows the total gain from this mutual fund, including the amount I previously withdrew. Would it be possible to add a row for Realized Returns and Unrealized Returns to show the current gain after the initial withdrawal?

**Comment by CasualNarwhal** [comment]
A summary of realized and unrealized gains across all stocks and funds would be quite useful.  The unrealized gains are in play while the realized gains show the secured winnings.

Using the realized portion, you can also calculate a "break even" price for every symbol to know at what price your unrealized portion would start losing or gaining money.

**Comment by zypA13510** [comment]
Imagine this scenario: I bought 10 shares of X in Jan. for $100 total, sold all 10 shares for $50 in Apr. So a 50% loss is realized, and my position of X has been fully closed. Now, if I buy 10 shares of X again in Oct. for $60, and it is valued today at $80, I want Ghostfolio to show me +33%, not some negative number simply because I registered a loss months or even years ago.
